src/core/service.c

```diff
@@ -1646,15 +1646,10 @@ static Service *service_get_triggering_service(Service *s) {
          *
```

Comment:
Commit message:
- please prefix the title with `core: `,
- please explain the change done by the commit. `improve` is obvious.